_episodes/09-cmorization.md

```diff
@@ -0,0 +1,1053 @@
+> "obs4MIPs" or "ana4MIPs" datasets, for example, are specifically produced to
+> facilitate comparisons with model simulations.
+>
+> To reflect these different levels of access restriction, the ESMValTool team
+> has created a tier-system. The definition of the different tiers are
+> as follows:
+>
+> - **Tier1**: obs4MIPs and ana4MIPS datasets (can be used directly with the ESMValTool)
+> - **Tier2**: other freely available datasets (most of them will need some kind of
+>   cmorization)
+> - **Tier3**: datasets with access restrictions (most of these datasets will also
+>   need some kind of cmorization)
+>
+> These access restrictions are also the reason why the ESMValTool developers
+> cannot distribute copies or automate downloading of all observations and
+> reanalysis data used in the recipes. As a compromise we provide the
+> CMORization scripts so that each user can CMORize their own copy of the access
+> restricted datasets if they need them.
+>
+{: .callout}
+
+## Make a test recipe
+
+Now that we have downloaded the data, our end goal is to run a recipe that can
+load it and work with it. So we will make this recipe and try to run it. It will
+probably fail, but that's okay. As you will see, the error messages will come in
+handy. Moreover, the recipe will serve as a test case. Once we get it to run, we
+know we have completed our task.
+
+> ## Create a test recipe
+>
+> Create a simple recipe that loads the "FLUXCOM" data. It should include a
+> datasets section with a single entry for the "FLUXCOM" dataset with the correct
+> dataset keys, and a diagnostics section with two variables: gpp and gppStderr.
+> We won't need any preprocessors or scripts (set `scripts: null`), but you will
+> have to add a documentation section with a description, authors and
+> maintainer, otherwise the recipe will fail.
+>
+> Use the following dataset keys:
+>
+> - project: OBS6
+> - dataset: FLUXCOM
+> - type: reanaly
+> - version: ANN-v1
+> - mip: Lmon
+> - start_year: 2000
+> - end_year: 2000
+> - tier: 3
+>
+> Some of these dataset keys are further explained in the callout boxes in this
+> episode.
+>
+> > ## Answer
+> >
+> > Here's an example recipe
+> >
+> > ```yaml
+> > documentation:
+> >
+> >   description: Test recipe for FLUXCOM data
+> >
+> >   authors:
+> >     - kalverla_peter
+> >
+> >   maintainer:
+> >     - kalverla_peter
+> >
+> > datasets:
+> >   - {project: OBS6, dataset: FLUXCOM, mip: Lmon, tier: 3, start_year: 2000, end_year: 2000, type: reanaly, version: ANN-v1}
+> >
+> > diagnostics:
+> >   FLUXCOM:
+> >     description: Check that ESMValTool can load the cmorized FLUXCOM data without errors.
+> >     variables:
+> >       gpp:
+> >       gppStderr:
+> >     scripts: null
+> >
+> > ```
+> >
+> > Note: a recipe similar to this one is available under
+> > `~/path/to/ESMValTool/esmvaltool/recipes/examples/recipe_check_obs.yml`.
+> > That recipe includes checks all datasets for which CMORizers are available.
```

Comment:
I suggest replacing the path with its link e.g. [recipe_check_obs ](https://github.com/ESMValGroup/ESMValTool/blob/master/esmvaltool/recipes/examples/recipe_check_obs.yml.). Why users need to know about this recipe? Can we provide more information by referring users to some ESMValTool documentations? 

Another issue, we can add a tip here guiding users to have a look at the episode `Writing your worn recipe`.

Reply:
Removed the sentence with recipe_check_obs, and added the link to the episode on writing your own recipe 👍